docs/list.md

```diff
@@ -197,6 +197,17 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **How to sign up**: Sign in with [Github here](https://devict-hacktoberfest.herokuapp.com/) so they can track your PRs.
 - **Notes**: You need to create at least two PRs in any of the  Wichita developer community organizations listed [here](https://devict-hacktoberfest.herokuapp.com/)
 
+#### **devtron**
```

Comment:
I started to make the changes myself to this entry, but there were so many that I stopped. I would like you to please review your PR and format this Devtron entry differently. 

Devtron appears to be rewarding people for meaningful PRs during their several different events. This PR appears to only capture their Art-A-Thon event, not anything else.